### ISSUE TITLE
Easier modification of server via terminal gui or server.properties

### ISSUE THREAD

**Issue by EGAMatsu** [ClassiCube/MCGalaxy#672]
Alot of settings are scattered about which makes hosting from a separate server harder than it needs to be, considering i cant even load the gui :v
So i need to edit the property files directly but due to em being scattered, it makes it a pain

**Comment by UnknownShadow200** [comment]
What issues are you having with loading the GUI? What settings are scattered around? What exactly are you proposing should be changed?

**Comment by EGAMatsu** [comment]
The files for various elements of the server are scattered (as in .txt and .properties files) and I cant see a gui in a damn terminal lmao

**Comment by Goodlyay** [comment]
It's still not clear what you're proposing. For example the files that define ranks and server properties files are in the folders "ranks" and "properties" respectively, but do you want those to be combined into one folder or what?

**Comment by EGAMatsu** [comment]
one folder for ease of use from command line

**Comment by Goodlyay** [comment]
So are you saying you have no access to a file explorer? This sounds like a personal issue. Rearranging all the files into a less organized format and making that work with old server file structures is hardly a reasonable request for such an unusual case

**Comment by UnknownShadow200** [comment]
You're quite correct in that it's far from ideal how some settings files are in text folder - however, for the sake of maintaining backwards compatibility, I'm a bit reluctant to move them unless absolutely necessary

I'm happy to add some summary documentation on where the various settings file are and what each settings file is for though

**Comment by EGAMatsu** [comment]
Maybe can you make a Terminal Shell for it?
Like a text-based interface.

**Comment by UnknownShadow200** [comment]
Not sure what advantage there is in implementing that, compared to just editing using an existing text editor like nano

**Comment by rdebath** [comment]
A perhaps useful start for finding files.
https://github.com/UnknownShadow200/MCGalaxy/wiki/Data-and-configuration-files-overview
